handlers.py

```diff
@@ -153,23 +183,64 @@ def get_all_responses(self, message):
         return ['It looks like you tried to @ me but might have accidentally selected the role instead']
 
 
-class QueryGameHandler(MentionMessageHandler):
-    keywords = ['query games']
+class QueryHandler(MentionMessageHandler):
+    keywords = ['query']
 
     def get_all_responses(self, message):
-        return ['\n'.join([game.name for game in games])]
+        mention, remainder = split_by_first_mention(message)
+        found_keyword, remainder = self.split_string_by_keywords(remainder)
 
+        if message_pertains_to_all_games(remainder):
+            return ['\n'.join([game.name for game in get_known_games()])]
+        else:
+            attribute, game_name = remainder.split(' ')[:2]
+            game = lookup_game_by_name_or_alias(game_name)
+            attribute_display = {
```

Comment:
Arguably game should have this on it - no need to change it for this PR but thought I'd point out. That way if we change game properties (e.g. aliases -> alias_regexes) then all the changes are in one place.